docs/data/deployments.yml

```diff
@@ -221,3 +221,11 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Super-Data-Cluster
+  description: UK Science and Technology Facilities Council
```

Comment:
I think the key difference between our deployment and almost every other one is that users can deploy their own isolated Pangeo environment for their own group using a pointy-clicky interface. I'd like to capture that somehow.

"Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud."

Reply:
Fair. I've gone with:
`Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud. Supported by UK Science and Technology Facilities Council.`
All other descriptions state the organisation involved, hence why I've included STFC. Is it fair to say they *support* JASMIN?